src/main/lib/llm/openai.ts

```diff
@@ -0,0 +1,52 @@
+import { createOpenAI } from '@ai-sdk/openai'
+import { generateObject } from 'ai'
+import { z } from 'zod'
+import type { LLMAdapter, LLMGenOptions } from './types'
+import { stripMarkdownCodeBlocks } from './types'
+
+export class OpenAIAdapter implements LLMAdapter {
+  name = 'openai' as const
+  private client
+  
+  constructor(apiKey: string, baseUrl?: string) {
+    this.client = createOpenAI({
+      apiKey,
+      baseURL: baseUrl || undefined
+    })
+  }
+
+  async generateText({ prompt, model = 'gpt-4o', temperature, maxTokens }: LLMGenOptions): Promise<string> {
+    // Check if this is a SQL query generation prompt (contains schema information)
+    if (prompt.includes('table schema') || prompt.includes('SQL') || prompt.includes('postgres')) {
```

Comment:
This is going to result in a plethora of bugs and unexpected behavior in the future